perception/autoware_euclidean_cluster/README.md

```diff
@@ -48,14 +48,17 @@ This package has two clustering methods: `euclidean_cluster` and `voxel_grid_bas
 
 #### voxel_grid_based_euclidean_cluster
 
-| Name                          | Type  | Description                                                                                  |
-| ----------------------------- | ----- | -------------------------------------------------------------------------------------------- |
-| `use_height`                  | bool  | use point.z for clustering                                                                   |
-| `min_cluster_size`            | int   | the minimum number of points that a cluster needs to contain in order to be considered valid |
-| `max_cluster_size`            | int   | the maximum number of points that a cluster needs to contain in order to be considered valid |
-| `tolerance`                   | float | the spatial cluster tolerance as a measure in the L2 Euclidean space                         |
-| `voxel_leaf_size`             | float | the voxel leaf size of x and y                                                               |
-| `min_points_number_per_voxel` | int   | the minimum number of points for a voxel                                                     |
+| Name                                    | Type  | Description                                                                                      |
+| --------------------------------------- | ----- | ------------------------------------------------------------------------------------------------ |
+| `use_height`                            | bool  | use point.z for clustering                                                                       |
+| `min_cluster_size`                      | int   | the minimum number of points that a cluster needs to contain in order to be considered valid     |
+| `max_cluster_size`                      | int   | the maximum number of points that a cluster needs to contain in order to be considered valid     |
```

Comment:
I think currently, this parameter stands for the maximum number of voxels? shall we rename it for more understandable?